packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -173,12 +174,22 @@ export class FrontendPluginImpl {
       };
     }
 
-    const authPlugin = ctx.configOfOtherPlugins.get(DependentPluginInfo.RuntimePluginName);
-    if (authPlugin) {
-      runtimeEnv = {
-        endpoint: authPlugin.get(DependentPluginInfo.RuntimeEndpoint) as string,
-        startLoginPageUrl: DependentPluginInfo.StartLoginPageURL,
-      };
+    if (isArmSupportEnabled()) {
```

Comment:
I assume after we support ARM provision for function, there will be a snippet like:
```
if (isArmSupportEnabled()) {
   // get function's output
}
```
Then I think it would be better to have an `originPostProvision` and a `postProvisioin` for ARM. And put the condition to index.ts. So that we can simply remove the `originPostProvision` when we decide to totally shift to ARM.

Reply:
It is a good suggestion. 
What we did now is to make a  prototype for arm support. It is a quick change.
After the prototype check, we can fork the arm support and origin flow.

Reply:
Ok for me.